src/main/jhipster/src/main/webapp/app/entities/germplasm/attribute/germplasm-attribute-type.ts

```diff
@@ -0,0 +1,4 @@
+export enum GermplasmAttributeType {
+    ATTRIBUTE = 'ATTRIBUTE',
+    PASSPORT = 'PASSPORT'
+}
```

Comment:
Later we can see if it's possible to use ids here and reuse eventually the same enum as the attribute-to-ontology branch: https://github.com/IntegratedBreedingPlatform/Workbench/commit/9aef8dcca67ce06d803fe208bdbb20eb79c7d4c9
But this should be ok for now, or let's wait until that one is merged and we can reuse it